docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md

```diff
@@ -364,6 +365,86 @@ For more information, check out Snowflake's official documentation on [the Snowf
 Repeat the previous steps to add additional Data Assets.
 
 
+</TabItem>
+<TabItem value="databricks">
+
+## Databricks SQL
+
+Connect GX to Databricks to access source data.
+
+### Prerequisites
+
+<Prerequisites requirePython = {false} requireInstallation = {false} requireDataContext = {false} requireSourceData = {null} requireDatasource = {false} requireExpectationSuite = {false}>
+
+- [An installation of GX set up to work with SQL](/docs/guides/setup/optional_dependencies/sql_databases/how_to_setup_gx_to_work_with_sql_databases)
+- Source data stored in a Databricks cluster
+
+</Prerequisites> 
+
+### Import GX and instantiate a Data Context
+
+<ImportGxAndInstantiateADataContext />
+
+### Determine your connection string
+
+The following code examples use a Databricks SQL connection string. A connection string connects GX to Databricks.
+
+The following code is an example of a Databricks SQL connection string format:
+
+```python
+my_connection_string = "databricks+connector://token:<token>@<host>:<port>/<database>?http_path=<http_path>"
+```
+
+### Create a Databricks SQL Datasource
+
+1. Run the following Python code to set the `name` and `connection_string` variables:
+
+    ```python
+    datasource_name = "my_databricks_sql_datasource"
+    ```
+
+2. Run the following Python code to create a Snowflake Datasource:
+
+    ```python 
+    datasource = context.sources.add_databricks_sql(
+        name=datasource_name, 
+        connection_string=my_connection_string,
+    )
+    ```
+
+### Connect to the data in a table (Optional)
+
+1. Run the following Python code to set the `asset_name`and `asset_table_name` variables:
```

Comment:
Do we mention anywhere that if `table_name` is not set, that it will use the asset `name`?

Reply:
hmm I don't know tbh - could follow up with that info if we desire (or update the docstring)